src/main/java/resources/TwitterRequestHandler.java

```diff
@@ -0,0 +1,54 @@
+package resources;
+
+import api.ErrorMessage;
+import api.ExceptionHandler;
+import api.StatusList;
+import api.Message;
+
+import twitter4j.*;
+import javax.ws.rs.*;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Path("/api/1.0/twitter")
+@Produces(MediaType.APPLICATION_JSON)
+public class TwitterRequestHandler {
+
+    private static Logger LOGGER = LoggerFactory.getLogger(ErrorMessage.class);
+
+    public TwitterRequestHandler(){}
+
+    @GET
+    @Path("/timeline")
+    public Response getTimeline(){
+        LOGGER.info("GET request at /api/1.0/twitter/timeline was triggered");
```

Comment:
ideally you don't explicitly log the version in plaintext as the route could change and this message would no longer be accurate